templates/android/library/src/main/java/io/package/Client.kt.twig

```diff
@@ -364,16 +364,22 @@ class Client @JvmOverloads constructor(
         var result: Map<*, *>? = null
 
         if (idParamName?.isNotEmpty() == true && params[idParamName] != "unique()") {
-            // Make a request to check if a file already exists
-            val current = call(
-                method = "GET",
-                path = "$path/${params[idParamName]}",
-                headers = headers,
-                params = emptyMap(),
-                responseType = Map::class.java,
-            )
-            val chunksUploaded = current["chunksUploaded"] as Long
-            offset = chunksUploaded * CHUNK_SIZE
+            try {
+                // Make a request to check if a file already exists
+                val current = call(
+                    method = "GET",
+                    path = "$path/${params[idParamName]}",
+                    headers = headers,
+                    params = emptyMap(),
+                    responseType = Map::class.java,
+                )
+                val chunksUploaded = current["chunksUploaded"] as Long
+                offset = chunksUploaded * CHUNK_SIZE
+            } catch (e: Exception) {
+                if (e.message != null && !e.message!!.contains("file could not be found")) {
+                    throw e
```

Comment:
Doesn't `call()` methods already throw these exceptions, is there any need to catch and throw here?